multiomics/README.md

```diff
@@ -0,0 +1,50 @@
+# Multiomics Datasets
+
+This repository contains a collection of publicly available multiomics datasets that have been slightly modified for use
+with the [PyKale](https://github.com/pykale/pykale) library.
+
+## 1. TCGA-BRCA
+
+The Cancer Genome Atlas Breast Invasive Carcinoma (TCGA-BRCA) is a multiomics dataset for subtype classification of 
+breast cancer (i.e., normal-like, basal-like, HER2-enriched, Luminal A, and Luminal B subtypes). The original version of
+this dataset can be obtained from the TCGA program through [Broad GDAC Firehose](https://gdac.broadinstitute.org/).
+
+Here, we provide the proprocessed version of TCGA-BRCA dataset in accordance with the **Preprocessing** section of the 
+[MOGONET](https://doi.org/10.1038/s41467-021-23774-w) [1] paper. This preprocessed dataset has been derived from the 
+[MOGONET repository](https://github.com/txWang/MOGONET/tree/main/BRCA) on GitHub. Three omics modalities are included in
+the collected dataset: gene expression RNA-seq (mRNA), DNA methylation (Meth), and miRNA expression (miRNA). A brief 
+description of these datasets is shown in Table 1.
+
+**Table 1**: Characteristics of the preprocessed BRCA multiomics dataset.
+
+|      Omics       | #Training samples | #Test samples | #Features |
+|:----------------:|:-----------------:|:-------------:|:---------:|
+| mRNA expression  |        612        |      263      |   1000    |
+| DNA methylation  |        612        |      263      |   1000    |
+| miRNA expression |        612        |      263      |    503    |
+
+
+## 2. ROSMAP
+
+Religious Orders Study/Memory and Aging Project (ROSMAP) is a multiomics dataset from Rush University for studying 
+Alzheimer’s disease (i.e., classification of Alzheimer’s disease patients vs. normal control). The original 
+version of this dataset can be obtained from [AMP-AD Knowledge Portal](https://adknowledgeportal.synapse.org/).
+
+Here, we provide the proprocessed version of ROSMAP dataset in accordance with the **Preprocessing** section of the 
```

Comment:
The same typo

Reply:
Corrected these typos in the revised version.